### DISCUSSION TITLE
Bio Page

### DISCUSSION THREAD

**Discussion by jacoby149** [chatscope/chat-ui-kit-react#67] [General] [answered]
Do you have any recommendations for packages for when you click the user circles, making the bio page?
Meaning showing the persons picture a little bigger, their phone number, something about them they wrote.
And also a page that lets you edit your own bio/phone number, and change your picture?

**Comment by supersnager** [comment, marked as answer]
Hey @jacoby149, 
I don't know a ready-made library for such purposes.
Personally, I use bootstrap and my own custom components. But it can be achieved by your favorite styling/components library.
